client/checkout/classic/upe-split.js

```diff
@@ -329,6 +329,41 @@ jQuery( function ( $ ) {
 		gatewayUPEComponents[ paymentMethodType ].upeElement = upeElement;
 	};
 
+	function togglePaymentMethodForCountry(
+		paymentMethodType,
+		supportedCountries
+	) {
+		const billingCountry = document.querySelector( '#billing_country' )
+			.value;
+		const affirmContainer = document.querySelector(
```

Comment:
And if affirmContainer is not there (Affirm disabled)?

Reply:
Also `affirmContainer` name is misleading at this point.

Reply:
I missed the naming, will update.

As for the case when it's not there. It's not possible. 
```JS
const upeDOMElements = $( '.wcpay-upe-element' );
for ( let i = 0; i < upeDOMElements.length; i++ ) {
	const upeDOMElement = upeDOMElements[ i ];
	const paymentMethodType = $( upeDOMElement ).attr(
		'data-payment-method-type'
	);
				```
				
If it's not on the page, then we won't try to restrict it. Simple as that.

Reply:
Great, thanks!